app/src/main/res/values-de/strings.xml

```diff
@@ -19,26 +19,26 @@
     <string name="dialog_title_logout">Abmelden?</string>
     <string name="dialog_message_logout">Bist du sicher dass du dich abmelden möchtest?</string>
     <string name="dialog_title_reviews">Lade Bewertungen</string>
-    <string name="dialog_message_reviews">Bitte warten! Empfange Vorschauen.</string>
+    <string name="dialog_message_reviews">Bitte warten. Empfange Bewertungen.</string>
```

Comment:
You are most likely technically correct. However the exclamation mark feels very aggressive. That's why I changed it to a full stop. The rule that imperative sentences should end with exclamation marks is not really enforced in everday life.